doc/nightmare.md

```diff
@@ -1,5 +1,288 @@
-# Nightmare for FOLIO UI 
+# Nightmare for FOLIO UI
 
-To document
+* [Develop tests for a UI module](#develop-tests-for-a-ui-module)
+* [The test context object](#the-test-context-object)
+* [helpers](#helpers)
+* [namegen (helpers.js)](#namegen-helpersjs)
+* [openApp (helpers.js)](#openapp-helpersjs)
+* [Writing testable code]
```

Comment:
The link to this section is missing.  [md2toc](https://github.com/folio-org/okapi/blob/master/doc/md2toc) is your friend!